src/MainWindow.vala

```diff
@@ -40,6 +40,11 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
     private Gee.LinkedList<string> return_button_history;
     private Gtk.Label updates_badge;
     private Gtk.Revealer updates_badge_revealer;
+    private Gtk.Overlay overlay;
+    public Granite.Widgets.Toast toast;
```

Comment:
Does this need to be public anymore? It doesn't look to me like anything outside this class is accessing it.
```suggestion
    private Granite.Widgets.Toast toast;
```